RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+View.swift

```diff
@@ -122,6 +122,11 @@ extension NonFungibleTokenDetails {
 						}
 						.padding(.vertical, .medium1)
 						.background(.app.gray5, ignoresSafeAreaEdges: .bottom)
+
+						IfLetStore(store.scope(state: \.hideAsset, action: \.child.hideAsset)) { store in
```

Comment:
hmm I thought one did not use `IfLetStore` from TCA anymore? [See migration guide 1.7](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/migratingto1.7/#Replacing-IfLetStore-with-if-let)

Cant we:
```swift
if let childStore = store.scope(state: \.hideAsset, action: \.child.hideAsset) {
    HideAsset.View(store: store).padding(...)
}
```
?

Reply:
mm I don't think so, the [method](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/store/scope(state:action:)-90255/) referred to in that guide returns a non-optional `Store<ChildState, ChildAction>`. 

Therefore, in this case it returns:
```Swift
Store<HideAsset.State?, HideAsset.Action>
```

while what we would need is:
```Swift
Store<HideAsset.State, HideAsset.Action>?
```

Reply:
We would be able to use this [method](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/store/scope(state:action:fileid:filepath:line:column:)-3yvuf) if we had this reducer updated to use `ObservableState`

Reply:
Right, ObservableState required